unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/WorldState.cs

```diff
@@ -128,5 +130,43 @@ public HashSet<Vector2Int> GetAllLoadedScenesCoords()
 
             return allLoadedParcelCoords;
         }
+
+        public List<GlobalScene> GetActivePortableExperienceScenes()
+        {
+            List<GlobalScene> activePortableExperienceScenes = new List<GlobalScene>();
+
+            foreach (var globalSceneId in globalSceneIds)
+            {
+                if (loadedScenes.ContainsKey(globalSceneId))
+                {
+                    GlobalScene peScene = loadedScenes[globalSceneId] as GlobalScene;
+                    if (peScene.isPortableExperience)
+                    {
+                        activePortableExperienceScenes.Add(peScene);
+                    }
+                }
+            }
+
+            return activePortableExperienceScenes;
+        }
+
+        public List<string> GetActivePortableExperienceIds()
+        {
+            List<string> currentSceneAndPortableExperiencesIds = new List<string>();
+
+            foreach (var globalSceneId in globalSceneIds)
+            {
+                if (loadedScenes.ContainsKey(globalSceneId))
+                {
+                    GlobalScene peScene = loadedScenes[globalSceneId] as GlobalScene;
+                    if (peScene.isPortableExperience)
+                    {
+                        currentSceneAndPortableExperiencesIds.Add(globalSceneId);
+                    }
+                }
```

Comment:
same as previous comment, we can use `TryGetValue` here

Reply:
Done!